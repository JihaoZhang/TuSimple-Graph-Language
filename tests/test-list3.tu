int main() {
	int i;
 	list @{int} l;
 	list @{int} l1;
 	new l1;
 	new l;
	print (l.length());
 	l.cancat(l1);
 	print (l.length());
 	for (i = 0; i < l.length(); i+=1) {
 		print(l.get(i));
 	} 
 	l.cancat(@{11,22,33});
 	l.remove(1);
 
 	for (i = 0; i < l.length(); i+=1) {
 		print(l.get(i));
 	} 
 	
 	l.pop();
 	
 	for (i = 0; i < l.length(); i+=1) {
 		print(l.get(i));
 	} 
 	
}