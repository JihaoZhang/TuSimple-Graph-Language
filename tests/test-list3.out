0
0
11
33
11